### PR TITLE
Archeo: Add home template

### PR DESCRIPTION
**Pull request by mikachan**
#### Changes proposed in this Pull Request:
This adds a home template to Archeo. It also adjusts the spacing in the patterns used on this template to match the comps more. 

- Removed the top margin on the list of posts patterns and the 'Layered images with headline' pattern
- Added top and bottom padding to the list of posts
- Added a variation of the list of posts pattern which adds the darker background and a title (I used a pattern so we could translate the title)

Screenshot taken without the English translations!

### CODE REVIEW

File: archeo/inc/patterns/simple-list-of-posts-with-background.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+	<div class="wp-block-group alignwide has-background-color has-foreground-background-color has-text-color has-background" style="padding-top:var(--wp--custom--spacing--medium);padding-bottom:var(--wp--custom--spacing--medium);margin-top:0px">
+		<!-- wp:heading {"level":3,"style":{"spacing":{"margin":{"bottom":"60px"}}},"fontSize":"medium"} -->
+		<h3 class="has-medium-font-size" style="margin-bottom:60px">' . __( 'Selected Writings', 'archeo' ) . '</h3>
+		<!-- /wp:heading -->
+		
+		<!-- wp:query {"queryId":3,"query":{"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":false,"perPage":3}} -->
+		<div class="wp-block-query"><!-- wp:post-template -->
+		<!-- wp:group {"className":"simple-list-of-posts","layout":{"type":"flex","allowOrientation":false}} -->
+		<div class="wp-block-group simple-list-of-posts"><!-- wp:post-title {"style":{"typography":{"lineHeight":"1.2","fontStyle":"normal","fontWeight":"100"}},"fontSize":"huge"} /-->
```
Comment (kjellr) [inline archeo/inc/patterns/simple-list-of-posts-with-background.php]: The post titles here need to be links — right now nothing is clickable. 😅

File: archeo/inc/patterns/simple-list-of-posts-with-background.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+		
+		<!-- wp:post-author-name {"style":{"typography":{"fontStyle":"normal","fontWeight":"100","lineHeight":"1.2"}},"fontSize":"medium"} /--></div>
+		<!-- /wp:group -->
+		<!-- /wp:post-template --></div>
+		<!-- /wp:query --></div>
```
Comment (kjellr) [inline archeo/inc/patterns/simple-list-of-posts-with-background.php:29]: I think this needs pagination. For now, let's just add prev/next links, with no page numbers. The text size can match our nav's text size.

**Review by kjellr — COMMENTED** [review]
This is looking good, thank you! 

Not sure if this is the best place to make this change, but the top header should have `var(--wp--custom--spacing--outer)` padding on the bottom to even it out. 

Also, there's a lot of extra padding above the `archeo/image-with-headline-description` pattern. That should have a top padding of `0` when placed here. (I think it would be fine to remove that padding from the pattern in general).

File: archeo/inc/patterns/simple-list-of-posts-with-background.php (hunk 1 above)
Comment (mikachan) [reply archeo/inc/patterns/simple-list-of-posts-with-background.php]: Oh yeah! 😅  I've added links now, although they automatically have an underline. Should we remove this just for this pattern?

image

File: archeo/inc/patterns/simple-list-of-posts-with-background.php (hunk 2 above)
Comment (mikachan) [reply archeo/inc/patterns/simple-list-of-posts-with-background.php:29]: Great ok, I've added pagination now too:

image

**Comment by mikachan** [comment]
I think this is ready for another review, thanks all!

**Review by scruffian — APPROVED** [review]
Code looks good to me. I think we can merge and let @kjellr iterate on the design in another PR if he notices more issues.

File: archeo/inc/patterns/simple-list-of-posts-with-background.php (hunk 1 above)
Comment (beafialho) [reply archeo/inc/patterns/simple-list-of-posts-with-background.php]: Thanks @mikachan! Can you remove the underline just for this pattern and have it be underlined on hover?

Comment (scruffian) [reply archeo/inc/patterns/simple-list-of-posts-with-background.php]: See https://github.com/Automattic/themes/pull/5573

**Pull request merged**